spec/models/politician_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+describe Politician do
+  describe 'validations' do
+    context 'invalid attributes' do
+      it 'is invalid without a name' do
+        politician = Politician.create(party: 'de', multiplier: 2, image: 'th')
+
+        expect(politician).to be_invalid
+      end
+
+      it 'has a unique name' do
+        politician = Politician.create(name: 'Dan', party: 'd', multiplier: 2, image: 't')
```

Comment:
This error message is incorrect. This test case is necessary to test uniqueness evaluation of the name column when creating new row in our Politicians table. FYI @Dpalazzari @njgheorghita @drod1000